desktop/libs/indexer/src/indexer/templates/importer.mako

```diff
@@ -1781,6 +1787,7 @@ ${ commonheader(_("Importer"), "indexer", user, request, "60px") | n,unicode }
       % endif
 
       self.inputFormat.subscribe(function(val) {
+        window.hueAnalytics.log('importer', 'source-type-selection/' + val);
```

Comment:
Perhaps this is the one I referred to above..

Reply:
Yep